running-locally.md

```diff
@@ -0,0 +1,38 @@
+Running locally for debug purposes
+==================================
+
+If the Scala Steward run is behaving incorrectly in some way, it may be worth
+running the Scala Steward code locally with [Java Remote debugging](https://www.jetbrains.com/help/idea/debugging-code.html)
+enabled.
+
+These instructions are adapted from Scala Steward's [docs/running.md](https://github.com/scala-steward-org/scala-steward/blob/main/docs/running.md),
+doing our best to duplicate the [parameters](https://github.com/scala-steward-org/scala-steward/blob/main/docs/help.md) which are [setup by the Scala Steward GitHub Action](https://github.com/scala-steward-org/scala-steward-action/blob/31bf30f6a29b33723e5b6d89a9a8f4cf23b05924/src/action/main.ts#L49-L71)
+when we run it in [our GitHub workflow](https://github.com/guardian/scala-steward-public-repos/blob/2c3ae97fe168c0887036fac81752b7d406653823/.github/workflows/public-repos-scala-steward.yml).
+
+1. Clone Scala Steward's codebase: [scala-steward-org/scala-steward](https://github.com/scala-steward-org/scala-steward).
+   We recommend [these alterations](https://github.com/scala-steward-org/scala-steward/compare/main...rtyley:scala-steward:temporary-tweaks-for-debugging) to the code:
```

Comment:
It's likely this branch will not exist forever. Should there be a more permanent home for the changes?

Reply:
Sure, would you prefer it if the branch was on a fork of [scala-steward](https://github.com/scala-steward-org/scala-steward) forked specifically into the Guardian GitHub org, rather than my own `rtyley` account? I could also be more specific in the name, like `DO_NOT_MERGE_OR_DELETE-tweaks-for-debugging` ?